### PR TITLE
hero section

### PR DESCRIPTION
**Pull request by tushargoyal07**
Defined Types for Navbar
Created Hero Section along with Marquee

### CODE REVIEW

File: components/Marquee/MarqueeData.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    {
+        title: "DATA BASE", 
+    },
+    {
+        title: "SERVER"
+    },
+]
+
+export const string2 : String2 = [
```
[the remark this answers is not in the document]
Comment (tushargoyal07) [reply components/Marquee/MarqueeData.ts]: Done

File: components/Marquee/types.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export type str ={
+    title: string
+};
+
+export type String1 = str[];
```
[the remark this answers is not in the document]
Comment (tushargoyal07) [reply components/Marquee/types.ts:5]: please check, changed

**Comment by tushargoyal07** [comment]
Integrated hamburger menu

**Review by byteoski — APPROVED** [review]
LGTM

**Pull request merged**